exporter/elasticsearchexporter/config.go

```diff
@@ -50,7 +50,9 @@ type Config struct {
 	// https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html
 	//
 	// This setting is required.
-	Index string `mapstructure:"index"`
```

Comment:
Please keep this option supported but deprecated. It should throw a warning if used

Reply:
got it.